product_docs/docs/biganimal/release/getting_started/creating_a_cluster/creating_a_dha_cluster.mdx

```diff
@@ -153,6 +165,25 @@ The following options aren't available when creating your cluster:
 
     For more information, see [Periodic maintenance](/biganimal/release/overview/updates).
 
+1. In the **Connections** section: 
+    
+   Enable **Read-only Workloads**. This feature configures a connection string you can use to perform read-only operations to shadow nodes (non-write leaders) to lighten the workload on the write leader and improve the PGD cluster's performance.
```

Comment:
Hi @neel5481 ! 

In a previous comment you suggested we add the info here that the customer can enable this feature for one or both data groups. 
I opted for adding a reference link at line 185 of this file, urgent customer to get more info from the overview page, and then I added that info there. WDYT?